static/js/settings_account.js

```diff
@@ -209,16 +209,64 @@ exports.set_up = function () {
     $("#show_api_key_box").hide();
     $("#api_key_button_box").show();
 
+    function show_reset_email_loader() {
+        loading.make_indicator($(".loading_indicator_spinner"), {text: 'Sending...'});
```

Comment:
This should be marked for translation.

Reply:
Yep strings marked with `i18n.t` function is marked for translation.